src/device-state/current-state.ts

```diff
@@ -234,6 +235,7 @@ export const startReporting = () => {
 
 	// If the state changes, report it
 	deviceState.on('change', doReport);
+	deviceState.on('change', downloadProgress);
```

Comment:
The startReporting function is for sending the current state to the cloud so flashing the LED given download progress is not relevant. I think moving it outside the function to the top of the file with a comment saying "This will indicate download progress via the device LED" is a good compromise.

Reply:
I think it depends on the definition of reporting. I'm personally OK with it being set here, I ultimately would like to get away from this type of event listener or at least I would like to have it being defined within a better API for device state.

Reply:
Yeah my thinking was that blinking the LED is also reporting information in a way. Added a comment to clarify what's doing but I'll leave it here unless you feel very strongly about it.